CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ find_package(Python COMPONENTS Interpreter Development REQUIRED)
 Python_add_library(_nvjitlinklib MODULE pynvjitlink/_nvjitlinklib.cpp WITH_SOABI)
 
 find_package(
-  CUDAToolkit 12.3 REQUIRED
+  CUDAToolkit 12.2.1 REQUIRED
```

Comment:
This version should allow conda-forge packages of 12.2u2 to be used. I'm not sure if this is the exact right pinning though, perhaps it reports itself as something like `12.2.140` to CMake and uses the third component from nvcc rather than using the update version. This should be checked before merging.

Reply:
Fixing this to 12.2.140 so we require 12.2u2 (12.2u1 might be sufficient but there's no harm in requiring the version on conda-forge since it's newer).